src/app/app.component.ts

```diff
@@ -1,9 +1,12 @@
 import { Component } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'Biomagnethic-App';
+
+
```

Comment:
do you have prettier? and is it formatting on save?

Reply:
Yes i have but git has its prettier

Reply:
but if you have prettier enable in vscode formatting on save and it will always format your files

Reply:
how i can do this

Reply:
in vscode: File -> preference -> settings and then look for format on save, click the checkbox, ready
![image](https://user-images.githubusercontent.com/11686968/216259885-f7d64fa3-7036-4a3c-aba0-6a4416fe4950.png)

Reply:
i have checked it